views/_partials/header.pug

```diff
@@ -27,7 +27,8 @@
                 li(class=(title === 'Integrations' ? 'active' : ''))
                     a(href='/integrations/' + themeQs) Integrations
                 li.dropdown
-                    a.dropdown-toggle(href='#', data-toggle='dropdown') Resources&nbsp;
+                    a.dropdown-toggle(href='#', data-toggle='dropdown') Resources
```

Comment:
This change didn't modify the display at all? I remember the `&nbsp;` being required, but perhaps I'm wrong.